sentry-core/src/main/java/io/sentry/core/EnvelopeReader.java

```diff
@@ -36,6 +36,7 @@
     while ((currentLength = stream.read(buffer)) > 0) {
       for (int i = 0; envelopeEndHeaderOffset == -1 && i < currentLength; i++) {
         if (buffer[i] == '\n') {
+          // TODO: IDE warns this is always true, is it?
```

Comment:
@bruno-garcia can you have a look at it?

Reply:
would be a different PR eh?

Reply:
by eye I can't see how. buffer gets modified on each call to `read`

Reply:
I really didn't check it, just got the warning from the IDE